src/components/Calculator.js

```diff
@@ -8,43 +8,48 @@ const Calculator = () => {
     setState(calculate(state, e.target.innerText));
   };
   return (
-    <div id="calculator">
-      <div id="input-area">
-        <span id="input-field">
-          {state.total}
-          {state.operation}
-          {state.next}
-        </span>
+    <div className="calc-page">
+      <div className="calc-header">
+        <h2>Let&apos;s do some math!</h2>
       </div>
-      <div id="button-area">
-        <div className="row row-1">
-          <button onClick={handleClick} type="button" className="calc-button first-line">AC</button>
-          <button onClick={handleClick} type="button" className="calc-button first-line">+/-</button>
-          <button onClick={handleClick} type="button" className="calc-button first-line">%</button>
-          <button onClick={handleClick} type="button" className="orange-button calc-button first-line">÷</button>
+      <div id="calculator">
```

Comment:
- [ ] **[OPTIONAL]** Your calculator unnecessarily covers almost the screen. It would be nice to reduce the size of your calculator to fit the screen.

![lufte-calc](https://user-images.githubusercontent.com/36542986/144227164-f5b6e255-a95c-4b6f-9a77-3e894fb1d42d.jpg)